src/com/massivecraft/massivetickets/cmd/CmdTicketsClose.java

```diff
@@ -0,0 +1,69 @@
+package com.massivecraft.massivetickets.cmd;
+
+import com.massivecraft.massivecore.MassiveException;
+import com.massivecraft.massivecore.command.requirement.RequirementHasPerm;
+import com.massivecraft.massivetickets.MassiveTickets;
+import com.massivecraft.massivetickets.Perm;
+import com.massivecraft.massivetickets.cmd.type.TypeMPlayer;
+import com.massivecraft.massivetickets.entity.MConf;
+import com.massivecraft.massivetickets.entity.MPlayer;
+
+import java.util.List;
+
+public class CmdTicketsClose extends MassiveTicketsCommand
+{
+	// -------------------------------------------- //
+	// INSTANCE
+	// -------------------------------------------- //
+	
+	private static CmdTicketsClose i = new CmdTicketsClose() { @Override public List<String> getAliases() { return MConf.get().aliasesOuterTicketsClose; } };
+	public static CmdTicketsClose get() { return i; }
+	
+	// -------------------------------------------- //
+	// CONSTRUCT
+	// -------------------------------------------- //
+	
+	public CmdTicketsClose()
+	{
+		// Parameters
+		this.addParameter(TypeMPlayer.getAny(), "player");
+		
+		// Requirements
+		this.addRequirements(RequirementHasPerm.get(Perm.CLOSE));
+	}
+	
+	// -------------------------------------------- //
+	// OVERRIDE
+	// -------------------------------------------- //
+	
+	@Override
+	public List<String> getAliases()
+	{
+		return MConf.get().aliasesInnerTicketsClose;
+	}
+	
+	@Override
+	public void perform() throws MassiveException
+	{
+		// Args
+		MPlayer ticket = this.readArg();
+		
+		// Force Sync
+		ticket.sync();
+		
+		// Is there a ticket?
+		if (!ticket.hasMessage())
+		{
+			msg("<white>%s <b>has not created a ticket.", ticket.getDisplayName(null));
+			return;
+		}
+		
+		// Apply
+		ticket.setModeratorId(null);
+		ticket.markAsDone(null);
+		
+		// Inform
+		MassiveTickets.alertModeratorsMsg("<white>%s <pink>closed <white>%s<pink>'s ticket.", msender.getDisplayName(null), ticket.getDisplayName(null));
```

Comment:
Please take a look at `CmdTicketsDone`:

```java
// Now mark it as done
ticket.markAsDone(msender);
		
// React
MConf.get().getDoneReaction().run(ticket.getModeratorId(), ticket.getId());
```

We can see that this new command is missing a "reaction", as well as a proper object oriented method that does the work such as `ticket.markAsDone` that could be called `ticket.markAsClosed` or something similar.

However, what is the point with adding a close? Could we not just support closing broken tickets with the existing "done" command? Is it necessary to add a new command? What is the benefit?